R/resp-body.R

```diff
@@ -77,6 +78,10 @@ resp_body_string <- function(resp, encoding = NULL) {
 #' @rdname resp_body_raw
 #' @export
 resp_body_json <- function(resp, check_type = TRUE, simplifyVector = FALSE, ...) {
+  if (env_has(resp$cache, "json")) {
```

Comment:
This completely ignores the arguments passed to `jsonlite::fromJSON()`. Usually, this shouldn't be an issue as it doesn't make much sense to use different argument. The exception is probably when requesting a new endpoint and you don't really know yet which arguments you want to use.

Reply:
Ooops, good point. Let me add that to the cache key.